swan-cern/files/swan_config_cern.py

```diff
@@ -46,38 +41,44 @@ async def get_swan_user_pod(self):
 
     async def _init_eos_secret(self):
         username = self.spawner.user.name
-        user_uid = self.spawner.user_uid
-        eos_secret_name ='eos-tokens-%s' % username
+        eos_secret_name = "eos-tokens-%s" % username
```

Comment:
I'd rather not have all these formatting changes, this is just going to create some noise and anyway the rest of files are not formatted.